pexpect/pty_spawn.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, command, args=[], timeout=30, maxread=2000,
         amount of data where you want to match. The searchwindowsize does not
         affect the size of the incoming data buffer. You will still have
         access to the full buffer after expect() returns.
+        When the keyword argument ``timeout`` is specified as a number,
+        (default: *30*), then :class:`TIMEOUT` will raise after the value
```

Comment:
TIMEOUT will *be raised*

Also, let's leave a blank line above this paragraph to separate it from the previous one.